src/components/SideBar/SideBar.tsx

```diff
@@ -167,6 +185,15 @@ const SideBar = ({
       setIsOptionClicked(false)
     }, 20)
   }
+  const [openChildrenItems, setOpenChildrenItems] = useState<number[]>([])
+
+  const toggleChildrenSubMenu = (index: number) => {
+    if (openChildrenItems.includes(index))
+      return setOpenChildrenItems(
```

Comment:
```suggestion
      const filteredChildrenItems  = openChildrenItems.filter((item) => item !== index)
      setOpenChildrenItems(filteredChildrenItems)
      return
```

And after, we don't need add return, just

```ts
setOpenChildrenItems([...openChildrenItems, index])
```